laptop/README.md

```diff
@@ -1,14 +1,28 @@
 # Sparkbox Laptop Maintenance
 
-## Wipe it clean
+Sparkbox employees are each given a laptop to do their work.
 
-Use the [Apple provided command line utility][apple-cli] to create a bootable drive. Use that drive to boot up the old Mac by holding down `OPT` during a restart. Once you've booted off the external drive, select Disk Utility and erase the internal Mac drive.
+These laptops aren't locked down. This gives our team the
+freedom to install whatever software and developer tools they need to be productive. With this freedom comes the responsibility to keep the device maintained and up to date, including complying with all [security requirements](../security/README.md).
 
-After that re-install the current and latest macOS.
+Sparkbox provides several tools and guidelines to help with laptop maintenance, as outlined below:
 
 ## Laptop Script
 
-Follow the instructions for the [Sparkbox Laptop] script.
+Sparkbox provides [a Laptop Script](https://github.com/sparkbox/laptop) which sets up the software, developer tools, and security practices we commonly use.
+
+Running this script is one of the first things we do on a new laptop. Its purpose is to provide a common foundation. Once it has been run, employees are free to install additional software and customize their development environment to meet their preferences.
+
+## Hardware Issues
+
+If you are having any hardware issues, let the office manager know and they will help you get them resolved.
+
+## Wiping a laptop
+
+When you are done using your laptop, you'll need to wipe it clean and reset it to a default state.
+
+To do this, use the [Apple provided command line utility][apple-cli] to create a bootable drive. Use that drive to boot up the old Mac by holding down `OPT` during a restart. Once you've booted off the external drive, select Disk Utility and erase the internal Mac drive.
```

Comment:
This is one way, another way is to hold down Cmd + R at boot to use the recovery volume to wipe the boot disk and reinstall macos. Note this recovery partition is usually for the version of macos that the computer shipped with. So this might not be ideal.

Reply:
Oh yeah, that's right. I liked how simple this approach was. It seems pretty hard to mess up.

Even if it reverts to an older version, you could just upgrade the OS through once the computer is reverted right?

If you think that approach is good enough for us to recommend, I can take a pass at writing those steps out.